### PR TITLE
BUGFIX: TMA-1022 Add more logs for case roll out without LDM change in master

### PR DESCRIPTION
**Pull request by phong-nguyen-duy**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/339/ : SUCCESS in 3s

File: lib/gooddata/lcm/actions/synchronize_ldm.rb [hunk 1]
```diff
@@ -89,6 +89,11 @@ def sync_segment_ldm(params, segment_info)
             maql_diff_params << :excludeFactRule if exclude_fact_rule
             maql_diff_params << :includeDeprecated if include_deprecated
             maql_diff = previous_master.maql_diff(blueprint: blueprint, params: maql_diff_params)
+            chunks = maql_diff['projectModelDiff']['updateScripts']
+            GoodData.logger.info "Synchronize LDM to clients will not proceed in mode \
+'#{params[:synchronize_ldm].downcase}' due to no change in new released LDM. \
```
Comment (danh-ung) [inline lib/gooddata/lcm/actions/synchronize_ldm.rb:94]: due to no LDM changes in the new master project.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/340/ : SUCCESS in 11s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/341/ : SUCCESS in 3s

File: lib/gooddata/lcm/actions/synchronize_ldm.rb (hunk 1 above)
Comment (phong-nguyen-duy) [reply lib/gooddata/lcm/actions/synchronize_ldm.rb]: updated

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/436/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-fossa-scanning-zuul-el7/7/ : SUCCESS in 37s (non-voting)

**Pull request merged**